concrete/src/Entity/Express/Entity.php

```diff
@@ -221,6 +221,22 @@ public function setDescription($description)
     }
 
     /**
+     * @param string $format
+     * @return string
+     */
+    public function getEntityDisplayDescription($format = 'html')
+    {
+        $value = t($this->getDescription());
```

Comment:
I don't think that the `t` is correct here. I'd remove the `getEntityDisplayDescription` method and simply use `h` when we output the description